doc/introduction/photonic_hardware.rst

```diff
@@ -134,25 +134,28 @@ Cloud simulator
 ---------------
 
 In addition to submitting jobs to be run on quantum hardware, it is also possible to run jobs on
-the Xanadu Quantum Cloud simulator Simulon. The process is very similar to running jobs on hardware. You
+simulators via the Xanadu Quantum Cloud. The process is very similar to running jobs on hardware. You
 will need to configure your account, as described above, and submit a job via the ``RemoteEngine``,
-using ``"simulon"`` as the target instead of a specific chip:
+using a simulator (a "simulon") as the target instead of a specific chip:
```

Comment:
`a "simulon"` sounds a bit strange, although I don't necessarily dislike it. 

Maybe it could be expanded into a separate sentence saying something along the line of "We refer to our remote simulators as 'simulon'".

Reply:
Ok, I added wording similar to this but kept it bracketed to avoid breaking the flow. Let me know what you think.

Reply:
Thanks @jswinarton. Looks better, I think! 🙂